snap/local/runtime-helpers/bin/edgex-cli-wrapper.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh -e
 
 # configuration.toml is required for the client to run
-if [ ! -f "$HOME/.edgex-cli/configuration.toml" ]; then
-    mkdir -p "$HOME/.edgex-cli"
-    cp "$SNAP/res/sample-configuration.toml" "$HOME/.edgex-cli/configuration.toml"
-    echo "Created $HOME/.edgex-cli/configuration.toml from sample configuration file."
-fi
+#if [ ! -f "$HOME/.edgex-cli/configuration.toml" ]; then
+#    mkdir -p "$HOME/.edgex-cli"
+#    cp "$SNAP/res/sample-configuration.toml" "$HOME/.edgex-cli/configuration.toml"
+#    echo "Created $HOME/.edgex-cli/configuration.toml from sample configuration file."
+#fi
 
 exec "$@"
```

Comment:
Is this file needed. If so, remove commented out code..

Reply:
not anymore. removed.